### ISSUE TITLE
Normal brake accumulator indicator wrong

### ISSUE THREAD

**Issue by oscnogbal** [saltysimulations/salty-747#87]
The normal brake accumulator on the simulator is at zero by default and only when you start engine 1 goes up to 3000 PSI. That is not correct. The brake accumulator is only for the hydraulic system 4, so only when the pressure in the system 4 raises should the accumulator indicator go up. With no hydraulic power in the system 4 the accumulator goes back to 0 after pressing the brake pedals 4-5 times.

**Comment by Lane-HOY** [comment]
Generally there is always brake pressure indicated, even with all electrical power OFF. This is the brake accumulator pressure, and as the park brake is set before shutting down engines, the pressure is maintained. Part of the "cockpit safety inspection" (done prior to putting on the battery in an cold and dark aircraft) is to check brake pressure normal.

**Comment by airifr** [comment]
> Generally there is always brake pressure indicated, even with all electrical power OFF. This is the brake accumulator pressure, and as the park brake is set before shutting down engines, the pressure is maintained. Part of the "cockpit safety inspection" (done prior to putting on the battery in an cold and dark aircraft) is to check brake pressure normal.

I don't think you know what you're writing.. Cold n dark means cold and dark, so you won't get any indication on any system, brake accumulator included, which by the way is POWERED by the ground handling bus, and like you say cold and dark the ground handling bus is NOT powered. And the brake accumulator doesn't show brake pressure it shows built-up system pressure, ie when no system pressure and parking brake NOT set, this shows accumulator precharge pressure. And never hear of safety inspection pre battery ON check brake pressure...??? Mind you generally the airplane is OFF brake when parked, is chocked. Unless for necessity to leave the parking brake on, the pressure will not hold for a day but several hours only then aircraft is free roll..
There is no brake pressure, there is hydraulic pressure that brakes use for operation, sys 4 (primary) sys 1 sys 2.. and the parking brake uses accumulator pressure hyd sys 4 in aux, that's what the pilots do during engine shutdown, momentarily place hyd 4 switch to aux to apply parking brake until the ground crew chock up the airplane, so that they can release parking brake afterwards if airplane is not to be towed..etc...etc..

**Comment by Lane-HOY** [comment]
The brake pressure gauge does not need electrical power as far as I know...
The brake pressure is rarely 0, it may not be 3000psi if there is a along ground time, but it holds some/most pressure.
If you shut down fully, and switch OFF the battery, you will still read the brake pressure. There is no pilot procedure to depressurize the system. There is no need for pilots.
When you turn the battery ON the pressure does not "reset" to zero (obviously).
If you take over an aircraft during a turnaround, there is brake pressure at 3000 psi from when brakes were set.
If cold and dark, you do not just walk into the cockpit and press the battery ON: you check first that certain critical things are OFF.
It's not at all stupid to look at the brake pressure, because it tells you if the chocks and/or the brakes are holding the aircraft in place.

**Comment by airifr** [comment]
> The brake pressure gauge does not need electrical power as far as I know...
> The brake pressure is rarely 0, it may not be 3000psi if there is a along ground time, but it holds some/most pressure.
> If you shut down fully, and switch OFF the battery, you will still read the brake pressure. There is no pilot procedure to depressurize the system. There is no need for pilots.
> When you turn the battery ON the pressure does not "reset" to zero (obviously).
> If you take over an aircraft during a turnaround, there is brake pressure at 3000 psi from when brakes were set.
> If cold and dark, you do not just walk into the cockpit and press the battery ON: you check first that certain critical things are OFF.
> It's not at all stupid to look at the brake pressure, because it tells you if the chocks and/or the brakes are holding the aircraft in place.

Stop writing...as far as you DON'T know... just stop writing .....

**Comment by ninjomcs** [comment]
be respectful to each other please

**Comment by Lane-HOY** [comment]
@ninjomcs Thanks for the comment ; I agree. We are all here contributing to make the Salty 748 the very best it can be.

@airifr , couple of three questions before I unpack things for you:
1. English not your first language, right? I'm asking to judge if you are misunderstanding, or if it's worse...
2. How many hours flight time do you have, what licenses do you have, and what types have you flown? Simulator doesn't count.
3. How much time do you have on the 747/744/748 in the real world? Passenger or jump seat does not count.
I'm ready to admit my mistakes, when I make them. Are you?

**Comment by airifr** [comment]
> @ninjomcs Thanks for the comment ; I agree. We are all here contributing to make the Salty 748 the very best it can be.
> 
> @airifr , couple of three questions before I unpack things for you:
> 
> 1. English not your first language, right? I'm asking to judge if you are misunderstanding, or if it's worse...
> 2. How many hours flight time do you have, what licenses do you have, and what types have you flown? Simulator doesn't count.
> 3. How much time do you have on the 747/744/748 in the real world? Passenger or jump seat does not count.
>    I'm ready to admit my mistakes, when I make them. Are you?

1. Who are you??
2. Who are you??
3. Who are you??

4. Get lost!!

**Comment by Lane-HOY** [comment]
I have thousands of hours PIC on the 744 (as well as other aircraft and helicopters).
I'm here to contribute positively to the Salty 748 Mod.

**Comment by ninjomcs** [comment]
@airifr I really don't like banning anyone from the discussion but if you continue having this type of attitude we might have to. I'm giving you one last chance. Please get familiar with our [code of conduct](https://github.com/saltysimulations/salty-747/blob/6499f92ed96135f102f570c35b89261a538a9090/CODE_OF_CONDUCT.md).

**Comment by airifr** [comment]
> @airifr I really don't like banning anyone from the discussion but if you continue having this type of attitude we might have to. I'm giving you one last chance. Please get familiar with our [code of conduct](https://github.com/saltysimulations/salty-747/blob/6499f92ed96135f102f570c35b89261a538a9090/CODE_OF_CONDUCT.md).

😴

**Comment by LaneTOGA** [comment]
I am going to reset, and restart the comment on the Brake Accumulator Pressure display on the gauge.
This is fact from the 744 Aircraft Maintenance Manual, the actual aircraft and the simulator.
1. Brake accumulator pressure is displayed electrics ON or OFF (the gauge light works with electrics ON only).
3. Brake accumulator pressure does not drop to zero on engine shut down or switching OFF all electrics.
4. Brake accumulator pressure is retained with Park Brake Set for a number of hours after shutdown.

**Comment by oscnogbal** [comment]
looking at the schematics I saw the brake accumulator gauge only shows the available pressure when power is  ON or the towing power switch is in the Battery position.

**Comment by LaneTOGA** [comment]
From the schematic, what is the power source for the brake accumulator gauge? Which electrical bus?

**Comment by oscnogbal** [comment]
28AC bus connected to the service bus. So whenever you have the service bus powered you get the indication. Could it be that you arrive to the F/D when the aircraft is on ground service and you think it is always on?

**Comment by LaneTOGA** [comment]
I didn't fly the 748, only 747 and 744. I can't find reference to the power source for the brake accumulator gauge.

On the 744 powered by the Ground Service Bus "makes sense", because it's power on the ground and in the air when AC Bus 1 is powered, and can be powered by the same source as the Ground Handling Bus if AC Bus 1 is unpowered (by pushing Ground Service switch at the L1 door).

If the aircraft was unpowered we'd always press that button to get lights to the upper deck and cockpit, even if the Battery Switch was still OFF. I cannot remember checking the pressure gauge in a cold and dark aircraft with no external power at all ...

**Comment by oscnogbal** [comment]
There we have our explanation then.
